public/js/showoff.js

```diff
@@ -102,10 +102,14 @@ function setupPreso(load_slides, prefix) {
 */
 }
 
-function loadSlides(load_slides, prefix) {
+function loadSlides(load_slides, prefix, reload) {
```

Comment:
load_slides and prefix are always loadSlidesBool and loadSlidesPrefix, global-level variables.  While we're fixing this up, do we want to remove those arguments?